src/locales/en_US.ts

```diff
@@ -675,6 +675,28 @@ const strings = {
   transaction_details_exchange_support: 'Exchange Support',
   transaction_details_exchange_support_request: '%s Support Request',
   transaction_details_fee_warning: 'High Network Fees',
+  transaction_details_swap: 'Swap Funds',
+  transaction_details_swap_order_cancel: 'Swap Order Canceled',
+  transaction_details_swap_order_post: 'Swap Order Opened',
+  transaction_details_swap_order_fill: 'Swap Order Filled',
+  transaction_details_stake: 'Stake Funds',
+  transaction_details_stake_order: 'Stake Order',
+  transaction_details_unstake: 'Unstake Funds',
+  transaction_details_unstake_order: 'Unstake Order',
+
+  transaction_details_swap_subcat_2s: '%1$s to %2$s',
+  transaction_details_swap_order_post_subcat_2s: '%1$s to %2$s Order Opened',
+  transaction_details_swap_order_fill_subcat_2s: '%1$s to %2$s Order Filled',
+  transaction_details_swap_order_cancel_subcat_2s: '%1$s to %2$s Order Canceled',
+  transaction_details_stake_subcat_1s: 'Stake %1$s',
+  transaction_details_stake_subcat_2s: 'Stake %1$s and %2$s',
+  transaction_details_stake_order_subcat_1s: 'Create Stake %1$s Order',
+  transaction_details_stake_order_subcat_2s: 'Create Stake %1$s and %2$s Order',
+  transaction_details_unstake_subcat_1s: 'Unstake %1$s',
+  transaction_details_unstake_subcat_2s: 'Unstake %1$s and %2$s',
+  transaction_details_unstake_order_subcat_1s: 'Create Unstake %1$s Order',
+  transaction_details_unstake_order_subcat_2s: 'Create Unstake %1$s and %2$s Order',
```

Comment:
I feel like these aren't really "categories", but more like notes. From an accounting point of view, I would expect something simple like "Expense:Staking" or "Exchange:Swap" for the category, and then details like "Trade XYZ token for ZZZ token" as a memo or note or something, but maybe I am wrong. I would ask @paullinator for a second opinion on this design.

Reply:
Yeah agreed, defaults for notes sound better. I'll sync up with him on this